test/node-api/test_threadsafe_function/binding.c

```diff
@@ -168,20 +181,31 @@ static void join_the_threads(napi_env env, void *data, void *hint) {
       napi_call_function(env, undefined, js_cb, 0, NULL, NULL));
   NAPI_CALL_RETURN_VOID(env, napi_delete_reference(env,
       the_hint->js_finalize_cb));
+  if (alt_ref) {
```

Comment:
Nit: Let's change this to `if (alt_ref != NULL)`

Reply:
Updated :)